pyproject.toml

```diff
@@ -91,8 +91,7 @@ lint = [
 ]
 test = [
     "pytest>=6.0",
-    "html5lib",
-    "defusedxml>=0.7.1", # for secure XML/HTML parsing
+    "defusedxml>=0.7.1",  # for secure XML/HTML parsing
```

Comment:
I think we could and should leave the previous line as-is here; it saves a few headaches while looking back through the line history -- and this is a TOML file, so double-space between statement and comment marker isn't a style we need to adhere to.

Reply:
It's to be consistent with the rest of the file

Reply:
Please remove the adjustment - I realize it's inconsistent.  This isn't the branch to address that.